src/pybind/mgr/orchestrator/module.py

```diff
@@ -1099,7 +1099,13 @@ def _apply_misc(self,
                 specs.append(spec)
         else:
             placementspec = PlacementSpec.from_string(placement)
-            assert service_type
+            try:
+                assert service_type
+            except:
+                out = ("Improper use of 'ceph orch apply'. No valid service type provided.\n"
+                       + usage + 'where <service_type> must be: '
+                       + '|'.join(list(map(lambda service: service.value, ServiceType))))
+                raise OrchestratorValidationError(out)
```

Comment:
```suggestion
            if not service_type:
                raise OrchestratorValidationError(usage)
```

maybe just raise usage here?

Reply:
I'm basically just raising usage then listing the services out so this works for me